pyproject.toml

```diff
@@ -25,6 +25,7 @@ dependencies = [
     "requests",
     "semver",
     "setuptools",
+    "tomli",
```

Comment:
Could you squash this commit into the first one? Otherwise, the first commit is technically broken, it uses a dependency which isn't yet installed